lumicks/pylake/channel.py

```diff
@@ -231,6 +231,9 @@ def downsampled_like(self, other_slice, reduce=np.mean):
             The default is `np.mean`, but `np.sum` could also be appropriate for some
             cases, e.g. photon counts.
         """
+        assert isinstance(other_slice._src, TimeSeries), "This function is intended to downsample like a LF channel"
```

Comment:
```suggestion
        assert isinstance(other_slice._src, TimeSeries), "This function is intended to downsample like an LF channel"
```

Reply:
👍 

Actually decided to clarify the error a bit more.
`You did not pass a low frequency channel to serve as reference channel`
and
`Downsampled_like is only available for high frequency channels`

Reply:
Generally good to suggest a solution when reporting the error, so yeah. Much better :+1: